NExtends/Primitives/Generics/Generics.extensions.cs

```diff
@@ -227,5 +227,103 @@ select new
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static bool IsNullOrEmpty<TSource>(this IEnumerable<TSource> source) => source == null || !source.Any();
+
+        public static bool IsEquivalentTo<T>(this ICollection<T> expected, ICollection<T> actual, IEqualityComparer<T> comparer = null)
```

Comment:
Call directly Contains() il will be simpler :)

Reply:
You should ask the guy that added this in Lucca why he didn't use Contains( ) ;)

Reply:
I modified Contains() implementation in NExtends on monday, that's why... 
But now, in Generics.extensions.cs, IsEquivalentTo have pretty the same implementation than Contains()
we should factorise it

Reply:
OK, let's create an Issue for that, in order to avoid introducing regression in Lucca in case implementations are not strictly the same ;)